### ISSUE TITLE
training details

### ISSUE THREAD

**Issue by fcakyon** [fcakyon/sahi-benchmark#1]

**Comment by fcakyon** [comment]
Hello @LawVP-SP, for the visdrone results, COCO pretrained models are finetuned on Visdrone dataset with all 10 classes ignoring `ignore` and `other` classes.

Finetunings are performed for 24 epochs. All other training details can be found on the config files under https://github.com/fcakyon/sahi-benchmark/tree/main/mmdet_configs

**Comment by LawVP-SP** [comment]
@fcakyon Thank you very much for the clarifications. Could you publish the json files to obtain these results (mAP)?

**Comment by fcakyon** [comment]
@LawVP-SP you mean the dataset json files or results json files?

**Comment by fcakyon** [comment]
@LawVP-SP we did not provide any result on COCO classes in the paper. We provide the results for `xView` and `VisDrone` datasets.

**Comment by LawVP-SP** [comment]
> @LawVP-SP you mean the dataset json files or results json files?

The result json files. It would be interesting to obtain them in order to be able to filter the mAP obtained by class.

**Comment by fcakyon** [comment]
@LawVP-SP if you tell me which model and dataset you are interested in, I can upload the class-wise evaluation results for it 👍

**Comment by LawVP-SP** [comment]
Thank you very much @fcakyon 
I have been reviewing the classes and it would be interesting to know the values for the pedestrian, people, car and truck classes.

I would greatly appreciate if you could upload to the repository both the json of the GTs from the Visdrone dataset together with the json of the annotations obtained by the models (FCOS, VFNet and TOOD) after using your proposal.

**Comment by fcakyon** [comment]
@LawVP-SP I will try my best to upload all the result json files in a week (a bit busy with the deadlines currently)

**Comment by fcakyon** [comment]
@IvanGarcia7 you can find the fine-tuning configs here: https://github.com/fcakyon/sahi-benchmark/tree/main/mmdet_configs. 

Before fine-tuning, you can slice your COCO formatted dataset using sahi slicing utilities (https://github.com/obss/sahi/blob/main/docs/slicing.md) or use a crop augmentation during training (https://github.com/fcakyon/sahi-benchmark/blob/9477519fec806cfee7d95d0c168cd00ca7a928c6/mmdet_configs/xview_tood/tood_crop_300_500_cls_60.py#L91).

After, fine-tuning, use sahi inference: https://github.com/obss/sahi#framework-agnostic-slicedstandard-prediction

**Comment by IvanGarcia7** [comment]
Hello.

I have seen that another user has also asked for the files to generate the mAP, so one of my questions is solved.

Regarding the re-training, how exactly does it work, do you re-train with the outputs obtained after applying YOLOv5+SAHI or do you use the dataset dedicated to Visdrone training already annotated previously?

**Comment by IvanGarcia7** [comment]
@fcakyon 
So, for the results covered in your preprint, the fine-tuning is performed with the annotated dataset distributed by Visdrone to train, right?

**Comment by IvanGarcia7** [comment]
Great, thank you very much!

**Comment by fcakyon** [comment]
@IvanGarcia7 @LawVP-SP @lightyear12 all result and evaluation files + classwise error plots for xivew experiments are now publicly available at https://github.com/fcakyon/sahi-benchmark#xview-results

Will also upload all visdrone results soon 👍

**Comment by fcakyon** [comment]
visdrone result files are also present now: https://github.com/fcakyon/sahi-benchmark/blob/main/README.md#visdrone-results

**Comment by youngjae-avikus** [comment]
@fcakyon 

Did the inference result get better when you only did SF and didn't do SAHI?

**Comment by fcakyon** [comment]
@youngjae-avikus yes it increases results slightly but not as much as with SAHI, check Table 1 and Table 2 in the official ICIP22 paper: https://arxiv.org/pdf/2202.06934.pdf
